Sming/SmingCore/Rtc.cpp

```diff
@@ -0,0 +1,66 @@
+#include "../SmingCore/Rtc.h"
+
+RTC::RTC() {

```

Comment:
I think what Rtc.cpp/h files should be moved to Sming\SmingCore\Platform folder to keep same style (platform related stuff).